docs/Installation.md

```diff
@@ -99,7 +99,19 @@ Note: In case software-discovery-tool code is already checked out, do the follow
 
 (Alternatively, you can check with unittesting) <br />
 ```cd software-discovery-tool/src/tests``` <br />
-```pytest```
+
+If you run `pytest` as your logged user, it may give errors/warnings since you have given user `apache` ownership.
+
+* For Ubuntu (18.04, 19.04, 20.04):
+
+		sudo -u apache pytest
+
+* SLES (12 SP1, 12 SP2, 12 SP3):
+
+		sudo -u apache pytest
+		#if you are prompted with a password, then follow these:
+		sudo -u apache passwd #ignore all BAD PASSWORD warnings OR set a strong password
```

Comment:
I wouldn't document this, this is standard sudo behavior and we should expect that folks are familiar with the configuration of sudo on their system.

Reply:
Ah yeah, the password prompt came only in SUSE. Should I just mention that if you get a prompt, set a passwd for apache?

Reply:
No, I wouldn't mention it at all. The password it's looking for is actually *your* user password because it's your user triggering sudo. The only reason this fix works is because you end up putting your password in, so it now exists as long as your session is running. Setting a password on the apache user is generally not recommended.

Reply:
This issue is caused when we gave ownership to user `apache`. When using `sudo -u apache pytest` according to:
https://github.com/openmainframeproject/software-discovery-tool/pull/32#discussion_r652218678_
there's a password prompt only in SUSE. It's not my user but apache user :')

Reply:
Yeah, that's because of how SUSE is configured by default with sudo. In Ubuntu the sudo configuration is probably set up to not ask for their password.

And if you're running this command as your user, it really is your user password sudo is asking for, not apache (sudo is confusing in this regard, I know).

Reply:
@pleia2 :(
![image](https://user-images.githubusercontent.com/42383989/122594845-35fe0600-d085-11eb-8d9b-4f45c3a938fd.png)

Reply:
Hmm, ok. Well we really don't want to set a password for apache because it's a security risk and we do want the files to be owned by apache. Let's skip suggesting running pytest altogether as a mechanism for seeing if it's running. It's really only something that should be used in development anyway.

Reply:
I've removed the passwd part. If anyone faces any issues, they can raise an issue here. Nevertheless, I've removed all instructions